### PR TITLE
ArcGIS and SDMX port to jdk11

### PR DESCRIPTION
**Pull request by lmorandini**
Port of ArcGIS ReST API and SDMX datastore modules to Java 11.

### CODE REVIEW

**Comment by aaime** [comment]
Think @bradh might be interested in this one? :-D

File: modules/unsupported/arcgis-rest/pom.xml [hunk 1]
```diff
@@ -134,8 +131,8 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <source>1.8</source>
-                            <target>1.8</target>
+                            <source>11</source>
```
Comment (aaime) [inline modules/unsupported/arcgis-rest/pom.xml:134]: We still build targetting java 8, that's the minimum version, the goal is just to have it buliding and running on both versions.
That said, this is just a tool that needs to be called on demand? It's not a problem if it works only with java 11 I guess.

File: modules/unsupported/arcgis-rest/README.rst [hunk 2]
```diff
@@ -19,5 +19,5 @@ System tests
 
 Make sure to have all the necessary servwer certificates in a truststore.
```
Comment (bradh) [inline modules/unsupported/arcgis-rest/README.rst:20]: Another typo on this line: servwer -> server.

**Review by bradh — APPROVED** [review]
Only affects unsupported modules, seems fine.

File: modules/unsupported/arcgis-rest/pom.xml [hunk 3]
```diff
@@ -111,16 +103,21 @@
...
-            <artifactId>powermock-api-mockito</artifactId>
+            <artifactId>powermock-api-mockito2</artifactId>
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-cql</artifactId>
-            <version>23-SNAPSHOT</version>
+            <version>25-SNAPSHOT</version>
```
Comment (mprins) [inline modules/unsupported/arcgis-rest/pom.xml:113]: You should be able to use ${project.version}  here AFAIK

File: modules/unsupported/arcgis-rest/README.rst (hunk 2 above)
Comment (lmorandini) [reply modules/unsupported/arcgis-rest/README.rst]: Corrected.

File: modules/unsupported/arcgis-rest/pom.xml (hunk 1 above)
Comment (lmorandini) [reply modules/unsupported/arcgis-rest/pom.xml]: I made changes to the POMs to make them work with both JDKs.

File: modules/unsupported/arcgis-rest/pom.xml (hunk 3 above)
Comment (lmorandini) [reply modules/unsupported/arcgis-rest/pom.xml]: Done.

**Comment by lmorandini** [comment]
All the checks have now passed: could someone merge it?

**Pull request merged**